### DISCUSSION TITLE
UTM Tagging - Case sensitivity ?

### DISCUSSION THREAD

**Discussion by igrossma** [plausible/analytics#698] [Anything Else]
hi all,
playing around with UTM tags and I am little bit confused regards the right tagging style.

F.e. "Google" with the search integration is appearing in upperCase BUT if i integrate the UTM-Tag for googlemybusiness; it appears in lowercase. What is here the best practise to avoid later a lot of calculations or unclear reports for the customer. Same for facebook - should I always write the utm_source in UpperCase to appear in the right bucket @filter: ALL ?

![Bildschirmfoto 2021-02-09 um 16 34 13](https://user-images.githubusercontent.com/50171646/107388031-b1f3aa80-6af5-11eb-9549-744b34152169.png)
![Bildschirmfoto 2021-02-09 um 16 34 38](https://user-images.githubusercontent.com/50171646/107388038-b324d780-6af5-11eb-85eb-cd087137e693.png)

Best,
Ingo

**Comment by metmarkosaric** [comment]
thanks for reaching out @igrossma! 

UTM tags are case-sensitive so utm_source=Google and utm_source=google will show separately in your analytics dashboard. When the UTM tag is identical to the referral header we merge those clicks under one referral source.

Here are the referral header names for other popular websites so do aim to name your utm_source with identical names in order for both organic clicks and UTM clicks to be merged under the same referral source:

* Facebook
* GitHub
* Twitter
* LinkedIn
* Reddit
* Google

you can read more here https://plausible.io/blog/utm-tracking-tags

i hope this makes sense!

**Comment by igrossma** [comment]
Thanks Marko.

> Am 09.02.2021 um 17:44 schrieb Marko Saric <notifications@github.com <mailto:notifications@github.com>>:
> 
> https://plausible.io/blog/utm-tracking-tags <https://plausible.io/blog/utm-tracking-tags>

**Reply by metmarkosaric** [reply]
you're welcome!
